Update template-syntax.md (#302)
nuejs/nue: packages/nuejs.org/docs/template-syntax.md

</media-object>
```

> The slot element in the media object is replaced with the nested content in the loop. The nested content can contain anything: text, HTML tags, other custom components like product rating, commenting or product metadata.

Reviewer: ```suggestion
The slot element in the media object is replaced with the nested content in the loop. The nested content can contain anything: Text, HTML tags, other custom components like product rating, a comment section or product metadata.
```
WDYT about using `comment section`?

### Passing data with `:bind` { #bind }

> Bind directive makes every object property directly accessible to the component. Instead of writing `{ data.title }` inside the component, you can just write `{ title }`. This is particularly useful when looping components.

Reviewer: ```suggestion
The bind directive makes every object property directly accessible from the component. Instead of writing `{ data.title }` inside the component, you can just write `{ title }`. This is particularly useful when looping components.
```

Hm. I think we used different points of view. You from the parent component and me from the child component?

/>
```

> or you can use [:bind attribute](template-syntax.html#bind) to pass all the data at once:

Reviewer: ```suggestion
or you can use [:bind attribute](#bind) to pass all the data at once:
```
